x-pack/test/search_sessions_integration/config.ts

```diff
@@ -29,10 +29,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
 
     kbnTestServer: {
       ...xpackFunctionalConfig.get('kbnTestServer'),
-      serverArgs: [
-        ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
-        '--xpack.data_enhanced.search.sessions.enabled=true', // enable WIP send to background UI
-      ],
+      serverArgs: [...xpackFunctionalConfig.get('kbnTestServer.serverArgs')],
```

Comment:
Now that these functional tests do not need a specialized configuration, couldn't this file go away, and all the tests move to `x-pack/test/functional/apps` ?

Reply:
I thought that I'd like to leave it separately because I like how it tests the search_sessions integration feature within different apps. Otherwise, it will have to split between different `apps/` folders.
* It also has its own archives that rely on `shardDelay` agg and has tests that have to be skipped in case `shardDelay` is not available.  Easier to track that when it is all together

Reply:
I also like that we can assign ourselves as Codeowners on the whole folder. (at least until this stabilizes further)